Automattic/jetpack
# Simple Payments Block: refactor Edit component to function

## Proposed changes:

This PR refactors the Pay With Paypal block's Edit component to be a functional component. It's preparatory work for upgrading to block API to version 3.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack/issues/36765

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






- Spin up a test site with this branch
- If you run it locally, make sure to run `cd projects/plugins/jetpack && pnpm run build-extensions`
- Create a post
- Add the _Pay With Paypal_ block
- Make sure it behaves as it does in production
